commit f3a54316e658faa43cbf5af44f257e4e0f2270c0 (from 733453569ba14affe49070a9d41a20b42cce2e53)
Merge: 7334535 8fae3e1
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Fri Aug 19 09:30:59 2016 -0500

    Merge pull request #4652 from bgort/RCBugFix
    
    Add Rambo support/env to platformio.ini; properly maps extended pins

diff --git a/Marlin/platformio.ini b/Marlin/platformio.ini
old mode 100644
new mode 100755
index fb549db..8e66bd3
--- a/Marlin/platformio.ini
+++ b/Marlin/platformio.ini
@@ -42,3 +42,10 @@ platform = teensy
 framework = arduino
 board = teensy20pp
 build_flags = -I $BUILDSRC_DIR -D MOTHERBOARD=BOARD_BRAINWAVE_PRO -D AT90USBxx_TEENSYPP_ASSIGNMENTS
+
+[env:rambo]
+platform = atmelavr
+framework = arduino
+board = reprap_rambo
+build_flags = -I $BUILDSRC_DIR
+board_f_cpu = 16000000L

